packages/components/src/components/dropdown/_dropdown.scss

```
@@ -24,7 +24,7 @@
   .#{$prefix}--dropdown__wrapper--inline {
     display: inline-grid;
     align-items: center;
-    grid-gap: rem(24px);
+    grid-gap: 0 rem(24px);
```

I think this is backwards. Should it be?
```suggestion
    grid-gap: rem(24px) 0;
```

---

@jnm2377 I think the syntax for gap is `<row-gap> <column-gap>` and we are setting 0 for the row gap to remove the vertical spacing caused by the listbox styles. Let me know if I'm misunderstanding 👀

---

ah ok ok. I think I was misunderstanding which one we were removing! I thought we were removing column gap. Gotcha gotcha.